src/main/scala/subsystem/RocketSubsystem.scala

```diff
@@ -53,6 +54,15 @@ trait HasRocketTiles extends HasTiles
 
     rocket
   }
+
+  abstract override def getSubsystemOMComponents(resourceBindingsMap: ResourceBindingsMap): Seq[OMComponent] = {
+    val rockets = getOMRocketCores(resourceBindingsMap)
+    val plics = plicOpt.map(_.device.getOMComponents(resourceBindingsMap))
+    val clints = clintOpt.map(_.device.getOMComponents(resourceBindingsMap))
+    rockets ++ plics.getOrElse(Nil) ++ clints.getOrElse(Nil)
+  }
+  def getOMRocketCores(resourceBindingsMap: ResourceBindingsMap): Seq[OMComponent] =
```

Comment:
Same could use-a-helper-method/only-called-once questions as above

Reply:
This is a collection point for the lower level OM*'s. 
I feel that a helper function would make the code less clear in this case.